packages/feedback/src/core/integration.ts

```diff
@@ -6,6 +6,8 @@ import type {
   FeedbackScreenshotIntegration,
   Integration,
   IntegrationFn,
+  OptionalFeedbackConfiguration,
```

Comment:
This feels very specific to be exported from types 🤔 I don't see us using this anywhere else? We should really only put stuff in the types package that _absolutely has to_ be there, as this becomes public API for other stuff too etc!

Reply:
Justification for exporting`OptionalFeedbackConfiguration` is because exported because it's part of the public api: it's the options for the integration constructor.

The other type `OverrideFeedbackConfiguration` is also part of the public api, it's used by all the public functions like `attachTo` & `createWidget` and so on. It's slightly different from the one above with regards to theme stuff that can't currently be overridden.

I touched the both in the pr to highlight the source

Reply:
gimme a suggestion and we'll do it!